client/src/components/About/AboutInformation/AboutInformation.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import './AboutInformation.scss';
+import { NavLink } from 'react-router-dom';
+import ContributorsList from './ContributorsList/ContributorsList';
+import StatisticCard from './StatisticCard/StatisticCard';
+import NumberOfMonumentsByStateBarChart from './NumberOfMonumentsByStateBarChart/NumberOfMonumentsByStateBarChart';
+
+/**
+ * Presentational component for the plain-text information displayed on the About Page
+ */
+export default class AboutInformation extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            showingAllContributors: false
+        };
+    }
+
+    handleMoreContributorsClick() {
+        const { showingAllContributors } = this.state;
+        this.setState({showingAllContributors: !showingAllContributors});
+    }
+
+    render() {
+        const { contributors, monumentStatistics } = this.props;
+        const { showingAllContributors } = this.state;
+
+        let mapNavLink = (
+            <NavLink onClick={e => {
+                e.preventDefault();
+                window.location.replace('/map');
+            }} to='/map' key='map'>click here</NavLink>
+        );
+
+        const readMoreContributorsLink = (
+            <div className='more-contributors-link'
+                 onClick={() => this.handleMoreContributorsClick()}>
+                Read More
+            </div>
+        );
+
+        const hideMoreContributorsLink = (
+            <div className='more-contributors-link hide-link'
+                 onClick={() => this.handleMoreContributorsClick()}>
+                Hide
+            </div>
+        );
+
+        return (
+            <div className='about-information-container'>
+                <h1>
+                    About Monuments + Memorials
+                </h1>
+                <p>
+                    <span className='font-italic'>Monuments + Memorials</span> is a crowd-sourced initiative developed at
+                    the Rochester Institute of Technology under the direction of Dr. Juilee Decker. Students in her spring
+                    2019 history course <span className='font-italic'>Monuments & Memory</span> began data collection by
+                    examining 26 states or territories in the US. The following fall, software engineering students in the
+                    Golisano College of Computer and Information Sciences built this web application to house data and to
+                    generate further interest and data collection. To view monuments and memorials, {mapNavLink}. To enter
+                    data, click here.
```

Comment:
This `click here` will have to be replaced with a link when we have a sign in page.

Reply:
Can you add a todo comment for this

Reply:
Will do.